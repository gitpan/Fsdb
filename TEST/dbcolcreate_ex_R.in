#fsdb -R C test
test: a

test: b

#   | dbfilealter -R C
