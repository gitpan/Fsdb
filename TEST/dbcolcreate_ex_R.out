#fsdb -R C test foo
test: a
foo: -

test: b
foo: -

#   | dbfilealter -R C
#   | dbcolcreate foo
